gtk/PrefsDialog.cc

```diff
@@ -1051,6 +1080,12 @@ Gtk::Widget* PrefsDialog::Impl::networkPage()
     w->set_tooltip_text(_("LPD is a tool for finding peers on your local network."));
     t->add_wide_control(row, *w);
 
+    t->add_section_title(row, _("Default Trackers"));
+    auto tv = new_text_view(TR_KEY_default_trackers, core_);
+    tv->set_tooltip_text(
+        _("a list of default trackers to be added to new public torrents (and existing ones, after restarting)"));
```

Comment:
```suggestion
        _("Trackers for public torrents to use automatically"));
```

(doesn't have to be this phrasing word-for-word, but something consistent with the other UI and the daemon CLI)